Tasks/app-store-promote/task.loc.json

```diff
@@ -16,6 +16,7 @@
   "version": {
     "Major": "1",
     "Minor": "178",
```

Comment:
Could you please bump task versions according to [steps](https://github.com/microsoft/app-store-vsts-extension/blob/master/docs/taskversionbumping.md)?

Reply:
@anatolybolshakov Yes, I just bumped it to `180.0`.